README.rst

```diff
@@ -64,6 +65,8 @@ To see full usage instructions for the ``start`` action, use ``-h``/``--help``:
       --node-disks map      Map of node names to block devices (default: None)
       --predictable         If specified, attempt to expose container ports to the
                             same port number on the host (default: False)
+      --register-gateway    If specified, register gateway with the cluster
+                            (default: False)
```

Comment:
Would it make sense to have it as default? Is there a case where having the "false" is needed?

Reply:
Right. Made that change in a new commit.